### ISSUE TITLE
Two phonology enhancements

### ISSUE THREAD

**Issue by uklomp** [Woseseltops/signbank#1190]
1. When we edit the phonology of a sign, we'd like to have 'neutral space' set as the default location, if that's possible? Its by far the most frequent location and it would save us some work. Important here is that the location is not set to 'neutral space' UNTIL people start working on the phonology. Another option that would already help is to have 'neutral space' at the top of the menu. 
2. When handedness is 2s, it would be easy if we only needed to select the handshape of the strong hand, and that this same handshape selection is then automatically copied to the weak hand (because 2s implies that both hands have the same handshape).

**Comment by susanodd** [comment]
From reading the description, it ought to be possible.

**Comment by uklomp** [comment]
@rosestamp how would you feel about these enhancements?

**Comment by uklomp** [comment]
Let's go ahead and implement this?

**Comment by susanodd** [comment]
Okay, great! The need for this became more obvious to me when I was making up stuff with the batch edit.

**Comment by susanodd** [comment]
I need to implement this separately, so it won't be done for Batch Edit until later.
There needs to be a kind of trigger set up for all the code, the Python code, the javascript code and the editable code.

Would it also be needed to have Handedness 1 clear the Weak Hand field?
I don't know how people typically enter the phonology. But probably nobody would first type in 2s plus a strong hand, then have the weak hand filled in automatically, and then decide it should be 1 handed.
(That use case scenario should not happen? But the software could make sure it doesn't?)
It would be a constraint that 2s implies strong and weak handshapes are the same.
And 1 implies weak is empty?
For the Neutral space, I can do both, so it's at the top and a default once editing. (Need to figure out the code so it's elegant and not a hack job. :)

In the past, we have used implicitly that if the Handedness is filled in, the gloss "has phonology" (this is used to exclude glosses without phonology from some of the internal searches, since otherwise they return thousands of results that need to be filtered, like for finding homonymns.)

**Comment by uklomp** [comment]
No, it's not needed to have Handedness = 1 clear the Weak Hand field because it's usually already empty. In other words, the use case scenario you describe does indeed not happen.
